commerce-theme-minium/commerce-theme-minium/src/css/minium/components/mini_cart.scss

```diff
@@ -28,11 +36,15 @@
 
 	&:not(.is-open) {
 		box-shadow: 0 0 0 $color-shadow;
-		transform: translateX(100%);
+		transform: translateX(getOpeningTranslation(right));
+	}
+
+	.rtl {
+		left: 0;
 	}
 
 	.rtl &:not(.is-open) {
```

Comment:
So, my understanding of scss is a little rusty, but I would expect these two selectors to end up transpiling to CSS as:
`.commerce-cart .rtl {...`
and
`.rtl .commerce-cart:not(.is-open) {...`

So,
1) Is that true? and
2) If that's true, that isn't what we want, is it? I expect `.rtl` to be on the `body` (or some other high-level element), so then `.commerce-cart .rtl` wouldn't be the proper selector. Right?

Reply:
@ethanbustad You're right, there's an & missing by mistake.